### DISCUSSION TITLE
[Wish] Indigo Clear Variations

### DISCUSSION THREAD

**Discussion by fuzenco** [zed-industries/zed#12786] [Feature Requests]
I don’t know who created the Indigo Clear theme but it’s crazy good. The translucency is amazing. Would it be possible to get alternate colors of that theme?

![ss 2024-06-07 at 2 17 21 PM](https://github.com/zed-industries/zed/assets/173683/0d1ab956-5525-4cbb-9130-d1fbce6492b8)

**Comment by Moshyfawn** [comment]
As it's [a community extension](https://github.com/p3rception/Indigo-zed), you're better off creating an issue directly in their repository instead.

**Reply by Moshyfawn** [reply]
Side note, there's a GitHib icon button next to every extension that will lead you directly to the source code of that extension.

**Reply by fuzenco** [reply]
Thank you. Posted over there.
